04/app.js

```diff
@@ -1,5 +1,52 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+	console.log('DOM');
+	const formPanel = document.querySelector('.form');
+	formPanel.addEventListener('submit', showWeatherInfo);
+}
+
+const latitudeInput = document.querySelector('.form__field--lat').value;
+const longitudeInput = document.querySelector('.form__field--lng').value;
+
+let weatherLat = document.querySelector('.weather__lat');
+let weatherLng = document.querySelector('.weather__lng');
+
+let weatherSummary = document.querySelector('.weather__summary');
+let weatherTemperature = document.querySelector('.weather__temperature');
+
+function showWeatherInfo(e) {
+	e.preventDefault();
+
+	if (coordinatesValidation(latitudeInput, longitudeInput)) {
+		enterCoordinates();
+		queryPromise();
+	} else {
+		alert('Wrong input!');
+	}
+}
+
+function coordinatesValidation(latitudeInput, longitudeInput) {
+	const regexExp = /^((\-?|\+?)?\d+(\.\d+)?),\s*((\-?|\+?)?\d+(\.\d+)?)$/gi;
+	const str = `${latitudeInput},${longitudeInput}`;
+	return regexExp.test(str);
+}
```

Comment:
Jeśli tworzysz walidację to musisz poinformować jaka jest poprawna - nie sądzę, aby mu się chciało kombinować.
PS. Wpisanie samych liczb też powinno być ok :P